Description: fix compile error regarding linux2.l_i_reserved2 deprecation
  See git commit 89efc88e65136ece22708cc28ec4124a33feeecd
  "libext2fs: add metadata checksum and snapshot feature flags"
  in upstreams e2fsprogs repository
Forwarded: yes
Author: Michael Prokop <mika@debian.org>
Last-Update: 2011-01-11

--- a/src/extundelete.cc
+++ b/src/extundelete.cc
@@ -2611,5 +2611,6 @@ void parse_inode_block(struct ext2_inode *inode, const char *buf, ext2_ino_t ino
   //FIXME: need to change behavior depending on the fs operating system
 	inode->osd2.linux2.l_i_uid_high = le16_to_cpu( (uint16_t *) &inodebuf[item*58] );
 	inode->osd2.linux2.l_i_gid_high = le16_to_cpu( (uint16_t *) &inodebuf[item*60] );
-	inode->osd2.linux2.l_i_reserved2 = le32_to_cpu( (uint32_t *) &inodebuf[item*62] );
+	inode->osd2.linux2.l_i_checksum_lo = le16_to_cpu( (uint16_t *) &inodebuf[item*62] );
+	inode->osd2.linux2.l_i_reserved = le16_to_cpu( (uint16_t *) &inodebuf[item*63] );
 }
